src/gui/tray/Window.qml

```diff
@@ -392,7 +392,7 @@ Window {
                             Layout.alignment: Qt.AlignRight
                             verticalAlignment: Qt.AlignCenter
                             Layout.margins: Style.accountDropDownCaretMargin
-                            source: "qrc:///client/theme/white/caret-down.svg"
+                            source: Style.dropDownCaretImageSource

```

Comment:
OK, I think you're going to hate me for the following comment: I wouldn't have pushed it so much on the C++ side in that case.

Indeed unlike the previous case @allexzander had where it was clearly tied to the branding business logic in that case it's more of a pure GUI concern of "please pick something compatible with a given background color".

So instead I think I'd "just" have done the following:
  * Make sure `isDarkColor` is reachable from the QML side
  * introduced a `headerIconFlavor` string property in `Style` which would pick `"white"` or `"black"` depending on what `isDarkColor` says about `ncBlue`
  * then use `Style.headerIconFlavor` in the URLs here

Reply:
See #2698 :)